spec/requests/posts_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :request do
+  describe 'GET /posts' do
+    it 'renders index template with correct placeholder text for post' do
+      get '/posts'
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:index)
+      expect(response.body).to include('Here are lists of posts')
+    end
+  end
```

Comment:
You are required to test the index page that renders all posts, which you have correctly done. 👏 
However, you are also required to also test the show page for a single post as well as a single user. Kindly add them.

![image](https://user-images.githubusercontent.com/27709832/223282427-5495ed32-baac-4cc4-a121-45ff67124826.png)